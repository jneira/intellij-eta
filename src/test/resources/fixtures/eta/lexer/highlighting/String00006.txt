ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('String00006')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvarid ('s')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITstring ('"')
ITstring ('\\')
ITstring ('\"')
ITstring ('"')
WHITE_SPACE ('\n\n')
ITvarid ('foo')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('undefined')
WHITE_SPACE ('\n')
