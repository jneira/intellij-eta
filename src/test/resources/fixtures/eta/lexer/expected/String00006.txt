ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('String00006')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvocurly ('')
ITvarid ('s')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITstring ('"\\\""')
WHITE_SPACE ('\n\n')
ITsemi ('')
ITvarid ('foo')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('undefined')
WHITE_SPACE ('\n')
ITsemi ('')